lib/modules/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib }:
```

Comment:
I'm starting to wonder if this should be called `types` instead of `modules`.

Reply:
Yes I was thinking the same. Although I won't do it right now, it'll cause too many conflicts. I can do that afterward...

Reply:
Resolving this then.